src/platform/plugins/shared/dashboard/server/api/transforms/out/transform_references_out.ts

```diff
@@ -8,70 +8,11 @@
  */
 
 import type { SavedObjectReference } from '@kbn/core/server';
-import { tagSavedObjectTypeName } from '@kbn/saved-objects-tagging-plugin/common';
-import type { DashboardState, DashboardPanel } from '../../types';
-import { isDashboardSection } from '../../../../common';
-import { embeddableService } from '../../../kibana_services';
-import { getPanelIdFromReference } from '../../../../common/reference_utils';
 
-export function isSearchSourceReference(reference: SavedObjectReference) {
-  return reference.name.startsWith('kibanaSavedObjectMeta');
+export function isLegacyControlGroupReference(reference: SavedObjectReference) {
+  return reference.name.startsWith('controlGroup_');
```

Comment:
Nit. We define this `controlGroup_` prefix as a local variable in several locations. We could consider resolving all of these to a single constant. Maybe not a big deal since we'll remove them most of them eventually.

- src/platform/plugins/shared/dashboard/common/reference_utils.ts
- src/platform/plugins/shared/dashboard/server/dashboard_saved_object/migrations/migrate_extract_panel_references/dashboard_container_references.ts
- src/platform/plugins/shared/controls/public/controls/data_controls/reference_name_utils.ts